godet.go

```diff
@@ -420,6 +427,35 @@ func (remote *RemoteDebugger) Reload() error {
 	return err
 }
 
+// CaptureScreenshot takes a screenshot, uses "png" as default format.
+func (remote *RemoteDebugger) CaptureScreenshot(format string, quality int, fromSurface bool) ([]byte, error) {
```

Comment:
I was thinking of passing the output filename instead of the format, and save the result to the file
(then the format can be derived from the filename).